src/self-help/azext_self_help/aaz/latest/self_help/diagnostic/_create.py

```diff
@@ -0,0 +1,378 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "self-help diagnostic create",
+    is_preview=True,
+)
+class Create(AAZCommand):
+    """Creates a diagnostics that will help you troubleshoot an issue with your azure resource.
+    """
```

Comment:
Could you add examples in aaz-dev-tools and regenerate the code?

Reply:
I actually added examples for all the commands. I do not see any changes pending in my local.

Reply:
The examples seems to be in aaz repo.  
https://github.com/Azure/aaz/pull/176/commits/c7c6c34e81c73b9373908ae43c199959142ca27f

PR for AAZ repo: https://github.com/Azure/aaz/pull/176